pkg/config/config.go

```diff
@@ -125,6 +125,21 @@ const (
 	standaloneConfigFileName = ".regal.yaml"
 )
 
+func FromPath(path string) (Config, error) {
```

Comment:
Is there a way we might adjust FindConfig to first check if the path is a valid config file before searching?

Reply:
But the path provided to FindConfig isn't a config file, but just any directory, no?